lib/error/bulkSaveIncompleteError.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Module dependencies.
+ */
+
+'use strict';
+
+const MongooseError = require('./mongooseError');
+
+
+/**
+ * If the underwriting `bulkWrite()` for `bulkSave()` succeeded, but wasn't able to update or
+ * insert all documents, we throw this error.
+ *
+ * @api private
+ */
```

Comment:
i think this block should be on `constructor` instead of on `class` like the other errors

Reply:
We're actually mixed on whether the comment is on the class or the constructor. For example, `MongooseBulkWriteError` and `CastError` have the comment on the class, but `DivergentArrayError` and `InvalidSchemaOptionError` have the comment on the constructor. We'll make this more consistent in a future PR.